easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenEXR'
+version = '2.3.0'
+
+homepage = 'http://www.openexr.com/'
+description = """OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic
+ for use in computer imaging applications"""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = ['https://github.com/openexr/openexr/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['8243b7de12b52239fe9235a6aeb4e35ead2247833e4fbc41541774b222717933']
+
+builddependencies = [('CMake', '3.12.1')]
+
+sanity_check_paths = {
```

Comment:
I find this hard to read, how about
```
sanity_check_paths = {
    'files': ['lib/lib%s-2_3.%s' % (x, SHLIB_EXT) for x in
              ['Half', 'Iex', 'IexMath', 'IlmImf', 'IlmImfUtil', 'IlmThread', 'Imath']] +
             ['bin/exr%s' % x for x in
              ['envmap', 'header', 'makepreview', 'maketiled', 'multipart', 'multiview', 'stdattr']],
    'dirs': ['include/OpenEXR', 'share'],
}
```

Reply:
Please also avoid hardcoding version numbers, use `%(version_major)s` and `%(version_minor)s`:

```python
sanity_check_paths = {
    'files': ['lib/lib%s-%%(version_major)s_%%(version_minor)s.%s' % (x, SHLIB_EXT) for x in
              ['Half', 'Iex', 'IexMath', 'IlmImf', 'IlmImfUtil', 'IlmThread', 'Imath']] +
             ['bin/exr%s' % x for x in
              ['envmap', 'header', 'makepreview', 'maketiled', 'multipart', 'multiview', 'stdattr']],
    'dirs': ['include/OpenEXR', 'share'],
}